src/components/App/AppBar.test.tsx

```diff
@@ -34,17 +34,36 @@ it('selects the matching nav-tab', () => {
             label: 'foo',
             path: '/foo',
             showInNav: true,
+            component: React.Fragment,
+            exact: false,
           },
           {
             label: 'bar',
             path: '/bar',
             showInNav: true,
+            component: React.Fragment,
+            exact: false,
           },
         ]}
       />
     </MemoryRouter>
   );
 
+  await wait();
+  await wait();
+  await wait();
+  await wait();
+  await wait();
+  await wait();
+  await wait();
+  await wait();
+  await wait();
+  await wait();
+  await wait();
+  await wait();
+  await wait();
+  await wait(); // dont ask. Due to TabScroller doing stuff on requestAnimationFrame
```

Comment:
unfortunately, other things like `await wait(() => {}, {timeout: 10000})` did not work.